src/test/regress/expected/distributed_functions.out

```diff
@@ -618,6 +631,13 @@ SELECT create_distributed_function('eq_with_param_names(macaddr, macaddr)', dist
ype']::text[], ARRAY[]::text[],NULL, NULL), ('type', ARRAY['public.usage_access_type']::text[], ARRAY[]::text[],NULL, NULL), ('type', ARRAY['function_tests.dup_result']::text[], ARRAY[]::text[],NULL, NULL), ('function', ARRAY['public', 'usage_access_func']::text[], ARRAY['public.usage_access_type', 'integer[]']::text[],NULL, NULL), ('function', ARRAY['public', 'usage_access_func_third']::text[], ARRAY['integer', 'integer[]']::text[],0, 0), ('function', ARRAY['function_tests', 'notice']::text[], ARRAY['pg_catalog.text']::text[],NULL, NULL), ('function', ARRAY['function_tests', 'dup']::text[], ARRAY['pg_catalog.macaddr']::text[],0, 0), ('function', ARRAY['function_tests', 'eq_with_param_names']::text[], ARRAY['pg_catalog.macaddr', 'pg_catalog.macaddr']::text[],0, 0), ('function', ARRAY['function_tests', 'eq_mi''xed_param_names']::text[], ARRAY['pg_catalog.macaddr', 'pg_catalog.macaddr']::text[],NULL, NULL), ('function', ARRAY['function_tests', 'agg_sfunc']::text[], ARRAY['integer', 'integer']::text[],NULL, NULL), ('function', ARRAY['function_tests', 'agg_invfunc']::text[], ARRAY['integer', 'integer']::text[],NULL, NULL), ('function', ARRAY['function_tests', 'agg_finalfunc']::text[], ARRAY['integer', 'integer']::text[],NULL, NULL), ('aggregate', ARRAY['function_tests', 'my_rank']::text[], ARRAY['pg_catalog."any"']::text[],NULL, NULL), ('function', ARRAY['function_tests', 'agg_names_sfunc']::text[], ARRAY['function_tests.dup_result', 'function_tests.dup_result', 'function_tests.dup_result']::text[],NULL, NULL), ('function', ARRAY['function_tests', 'agg_names_finalfunc']::text[], ARRAY['function_tests.dup_result']::text[],NULL, NULL), ('aggregate', ARRAY['function_tests', 'agg_names']::text[], ARRAY['function_tests.dup_result', 'function_tests.dup_result']::text[],NULL, NULL), ('sequence', ARRAY['public', 'user_defined_seq']::text[], ARRAY[]::text[],NULL, NULL), ('role', ARRAY['postgres']::text[], ARRAY[]::text[],NULL, NULL), ('database', ARRAY['regression']::text[], ARRAY[]::text[],NULL, NULL), ('schema', ARRAY['public']::text[], ARRAY[]::text[],NULL, NULL), ('schema', ARRAY['mx_testing_schema']::text[], ARRAY[]::text[],NULL, NULL), ('schema', ARRAY['mx_testing_schema_2']::text[], ARRAY[]::text[],NULL, NULL), ('schema', ARRAY['mx_test_schema_1']::text[], ARRAY[]::text[],NULL, NULL), ('schema', ARRAY['mx_test_schema_2']::text[], ARRAY[]::text[],NULL, NULL), ('schema', ARRAY['test_schema_support']::text[], ARRAY[]::text[],NULL, NULL), ('schema', ARRAY['schema_colocation']::text[], ARRAY[]::text[],NULL, NULL), ('schema', ARRAY['function_tests']::text[], ARRAY[]::text[],NULL, NULL), ('schema', ARRAY['function_tests2']::text[], ARRAY[]::text[],NULL, NULL), ('extension', ARRAY['plpgsql']::text[], ARRAY[]::text[],NULL, NULL), ('collation', ARRAY['test_schema_support', 'english']::text[], ARRAY[]::text[],NULL, NULL)) SELECT citus_internal_add_object_metadata(typetext, objnames, objargs, distargumentindex::int, colocationid::int) FROM distributed_object_data;
```

Comment:
I was initially puzzled by why we have `pg_catalog` in the output. But, they are just the parameters, right? So, we do not mark any object in pg_catalog as distributed?

Reply:
Yes, those are just parameters.